### PR TITLE
Add even more detail to the comment

### PR DESCRIPTION
**Pull request by hensmi-amazon**
**Issue #:** None

**Description of changes:** Didn't find the comment entirely helpful and had a question on it so rewrote this section a bit.

**Testing:**
n/a
*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*

**Checklist:**

1. Have you successfully run `npm run build:release` locally?
y

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
n

3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
n

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by nainkunal933 — DISMISSED** [review]
Thank you for adding more documentation

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 ### Changed
+- Clarified a comment in `DefaultSimulcastUplinkPolicy`.
```
Comment (nainkunal933) [inline CHANGELOG.md:16]: I do not think you need a CHANGELOG update for changes that do not effect builders.

Comment (nainkunal933) [reply CHANGELOG.md:16]: @ltrung can you verify?

**Review by ltrung — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ltrung) [reply CHANGELOG.md:16]: Unfortunately as long as you touch src folder you need CHANGELOG :D

**Pull request merged**